src/pages/Explorer/Components/EditDialog.svelte

```diff
@@ -0,0 +1,100 @@
+<script context="module">
+  import { dialogs } from 'webkit/ui/Dialog'
+  import EditDialog from './EditDialog.svelte'
+
+  export function showEditDialog(itemData) {
+    dialogs.show(EditDialog, { itemData })
+  }
+</script>
+
+<script>
+  import Dialog from 'webkit/ui/Dialog'
+  import Toggle from 'webkit/ui/Toggle'
+  import { edit } from './api'
+
+  const MAX_TITLE_LENGTH = 25
+  const MIN_TITLE_LENGTH = 3
+
+  export let itemData
+
+  let closeDialog
+  let editedTitle = itemData.title
+  let editedDescription = itemData.description || ''
+  let editedPublic = !!itemData.isPublic
```

Comment:
```suggestion
  $: ({id, editKey, title, description, isPublic} = item)
```

Reply:
there is `item` property in `itemData` that we need to use that later, we cant rename it

Reply:
it does not allow me to edit values in the form

Reply:
Sorry, it should be a simple `let` statement, not a reactive one